public/app/features/trails/MetricSelect/api.ts

```diff
@@ -99,3 +104,8 @@ export async function getMetricNamesWithScopes(
     missingOtelTargets: false,
   };
 }
+
+function removeBrackets(input: string): string {
+  const match = input.match(/^\{(.*)\}$/); // extract the content inside the brackets
+  return match?.[1] ?? '';
+}
```

Comment:
In my utf8 support PR I did same kind of thing you did here. Let's see how they'll play together.